packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -69,14 +69,16 @@ export class AtomicRefineModal implements InitializableComponent {
   watchEnabled(modalStatus: ModalStatus) {
     const modalOpenedClass = 'atomic-modal-opened';
 
-    if (modalStatus === 'opened' || modalStatus === 'beingClosed') {
```

Comment:
This was causing a bug inserting facets multiple times

Reply:
Indeed, I [left a comment](https://github.com/coveo/ui-kit/pull/1215/files#r711253572) about this on @olamothe's PR. Thanks for fixing it,

Reply:
Oopsie ! Thanks !